src/views/ettersending/Ettersending.tsx

```diff
@@ -34,6 +45,18 @@ export function Ettersending(props: IProps) {
   const [bundlingComplete, setBundlingComplete] = useState(false);
   const [ettersendSoknad, ettersendSoknadStatus] = usePutRequest(`soknad/${uuid}/ettersend`);
 
+  const dokumentkravAvailableForEttersending = props.dokumentkrav.krav.filter(
```

Comment:
kanskje `availableDokumentkravForEttersending` hadde vært litt bedre navn? Samme gjelder for den under, og da blir det lettere å skille ut `avaliable..` og `unavaliable..`

Reply:
Fikset :)